docs/connectors/taps/github.rst

```diff
@@ -41,8 +42,29 @@ Example YAML for ``tap-github``:
     # ------------------------------------------------------------------------------
     db_conn:
       access_token: "<ACCESS_TOKEN>"            # Github access token with at least the repo scope
-      repository: "transferwise/pipelinewise"   # Path to one or multiple repositories that you want to extract data from
+      organization: "transferwise"              # The organization you want to extract the data from
```

Comment:
can you please change it to something generic open source and not company specific? Maybe `apache` or some other large OS repos is a better choice